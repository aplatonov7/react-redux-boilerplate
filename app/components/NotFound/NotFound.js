import React, { PropTypes } from 'react'

const NotFound = () => (
  <div>
    Not Found
  </div>
)

NotFound.propTypes = {

}

export default NotFound
