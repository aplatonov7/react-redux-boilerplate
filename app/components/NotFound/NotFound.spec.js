import React from 'react'
import { shallow } from 'enzyme'
import NotFound from './'

describe('NotFound component test', () => {
  const item = shallow(<NotFound />)

  it('Should render text', () => {
    expect(item.find('.root').contains('Not Found')).toBe(true)
  })
})
