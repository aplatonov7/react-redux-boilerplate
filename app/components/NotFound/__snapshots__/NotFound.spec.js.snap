exports[`NotFound component should match snapshot 1`] = `
<div>
  Not Found
</div>
`;
